### ISSUE TITLE
Mob Projectiles not hitting player

### ISSUE THREAD

**Issue by gloomyMoron** [scalda/Foolcraft_3#171]
## Modpack Information
- [x] I am running the latest version of FoolCraft
- [x] I am running FoolCraft via Curse/Twitch Launcher
- [x] I can reproduce this issue consistently
     - [x] In single player
     - [ ] In multiplayer
- [x] I have searched for this issue previously and it was either (1) not previously reported, or (2) previously fixed and I'm having the same problem.
- [ ] I am crashing and can provide my crash report(s)
- [ ] I have not altered the modpack [if you have, note the removed mods/added mods/changes in Additional Information]

## Issue Description
For awhile now, **mob projectiles (skeleton arrows, llama spit, blaze fireballs, witch potions, etc) have not been hitting me. In fact, they more often than not hit the mob that tried attacking me, miss entirely, or bounce off harmlessly. Even wearing no armor, no baubles, and holding nothing in my hand.** Oddly enough, if I use my bow (it has Multishot II on it) against an enemy that is close to me, my own arrows will hit me, so I can still, technically, be hit with projectiles. This is a lot like an old 1.9 bug, but that was supposedly fixed.

This has made it impossible to advanced down the "Focus Medium: Projectile" line in Thaumcraft 6 (unless I use a console command), since I can no longer be hit by blaze fireballs (in their case they miss entirely, even while standing perfectly still... even jumping INTO their fireball does no damage) nor llama spit (they spit at me but no damage is ever taken).

## Reproduce Steps

Play for awhile.
Get shot at by skeletons or other ranged mobs.
???
Take no damage

## Additional Information
I've only added one mo... two mods and a library to the game, but this was happening before I added those mods. First, I added JEHC (Just Enough HarvestCraft, since HarvestCraft grinder, presser, market, and shipping bin were all missing JEI integration). Then I added Hooked (a new grappling hook mod that just released and is pretty cool <.<) and it's required Library (Librarian Library).

**Comment by Mrbysco** [comment]
I've been noticing this too while playing. Also I guess you either posted this from phone or ignored the template.

**Comment by Mrbysco** [comment]
Accidentally clicked the wrong button. Ignore the assignment^

**Comment by gloomyMoron** [comment]
I did not see the template (I came straight to Issues without going to the 'main page' so I just didn't notice it). I'll edit it in.

Maybe it has something to do with Nutrition? I've been maxed for awhile and I think it started around the time I did that, but I can't really remember (I don't leave my little hole much right now >.>).

**Comment by swardhagen** [comment]
The exact same thing has been happening to me for a while, I thought it was a perk with the end dragon armor I made since I saw it happen about the time I made that but now I think we have the same bug!

**Comment by Kerellian** [comment]
I am seeing the same thing on a Multiplayer server. So far the only thing that i might be able to relate it to is having all the food types maxed out. Not sure if this helps but it is also affects player projectiles as well.

**Comment by bobbnu** [comment]
I have this issue too sometimes but reloading fixes after a few times, odd

**Comment by Revvilo** [comment]
From what I've experienced, it pointed to being given by the max level Toughness effect earned by having full nutrition, as @gloomyMoron has said.
Although there isn't any actual mention of this in the wiki about the potion effects, I really can't think of any other cause since it lined up so perfectly with me getting my food creation optimised, and thus my nutrition at full all the time.
Tho personally, I found it quite funny, just standing there, watching a skeleton shooting himself... 😄
*Edit:* Perhaps it is to do with the multiplication of the armour toughness attribute (by the Toughness effect), and then another mod causing the actual ricocheting of projectiles?
